rules/game_process.tex

```diff
@@ -75,6 +78,8 @@ \subsection{Robot States}
   \label{fig:robot_states}
 \end{figure}
 
+The referee must tell the GameController operator to switch to \texttt{initial} state once everyone is in place or that teams are falling under forfeit rule (see \cref{sec:forfeit}).
+The referee should wait between 10 to 120 seconds before announcing the transition to \texttt{ready}. 
```

Comment:
How about increasing the mininmum to 30 secs?  

Other opinions?

Reply:
I didn't want to introduce a different rule for CS and CC. I think this should be handled at referee meeting. I'm sure no one really gonna wait for 120 seconds doing nothing, but at the same time we don't want every initial state to be 30 seconds. I think this give more room to have the duration between 20-45 seconds.

Reply:
If we handle this during the referee meeting, then we could not give any time in the rules. Keeping it simpler.